crates/ruff_linter/src/rules/flake8_pyi/rules/custom_type_var_return_type.rs

```diff
@@ -270,12 +271,8 @@ fn add_diagnostic(checker: &mut Checker, function_def: &ast::StmtFunctionDef, re
         returns.range(),
     );
 
-    // See `replace_custom_typevar_with_self`'s doc comment
-    if in_stub {
-        if let Some(fix) = replace_custom_typevar_with_self(checker, function_def, returns) {
-            diagnostic.set_fix(fix);
-        }
-    }
```

Comment:
I moved the check for whether or not it's a stub file inside the body of `replace_custom_typevar_with_self()`, since that function already returns an `Option`. This means we don't have to have comments in two places :-)